happymonk-ai/Ckdir-frontend-
# fix success page height

What does this PR do?
• Fix height of success page for book demo form

How should this be manually tested?
• Clone this repo
• Check out this branch ft-homepage-update
• Run yarn install to install all packages
• Run yarn dev to start the development environment
• Navigate to "/register" fill the form correctly and test the changes.
